eclair-core/src/main/resources/reference.conf

```diff
@@ -162,6 +162,47 @@ eclair {
     }
 
     quiescence-timeout = 1 minutes // maximum time we will stay quiescent (or wait to reach quiescence) before disconnecting
+
+    channel-update {
+      // Balance thresholds at which to update the maximum HTLC amount
+      // Must be in increasing order.
+      balance-thresholds = [{
```

Comment:
How to disable it if the user doesn't want to leak their liquidity?

Reply:
You would simply set this to an empty array (`balance-thresholds = []`). I think it's worth mentioning that explicitly in the comment, and in the release notes!

Reply:
Done